app/presenters/document_presenter.rb

```diff
@@ -0,0 +1,116 @@
+##
+# Base class for Blacklight document presenters
+class DocumentPresenter < Europeana::Blacklight::DocumentPresenter
+  delegate :t, to: I18n
+
+  def render_field_value(value = nil, field_config = nil)
```

Comment:
Cyclomatic complexity for render_field_value is too high. [8/6]<br>Assignment Branch Condition size for render_field_value is too high. [17.72/15]<br>Method has too many lines. [11/10]<br>Perceived complexity for render_field_value is too high. [8/7]